iree/base/CMakeLists.txt

```diff
@@ -12,8 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# bazel_to_cmake: DO NOT EDIT (tracing rule move)
```

Comment:
Hmm.. so to switch a file from manually generated to autogenerated, what's needed?

A) Delete the file and run the script
B) Add `# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py` somewhere in the file

Both seem non-obvious, especially compared to removing a `# bazel_to_cmake: DO NOT EDIT` comment. Might be a fair trade-off though, considering the other benefits of this change.

Reply:
I think "delete the file" isn't too bad. I wouldn't expect someone to manually add the autogenerated tag.

Reply:
SG. Let's go with this then.